README.md

```diff
@@ -53,91 +41,54 @@ We are strongly opinionated about adding new variables/tokens to the colors. Aft
-
-### Join us in our [Discord Chat](https://discord.gg/qcynkd4Edx) here.
-
----
+Join us in our [Discord Chat](https://discord.gg/qcynkd4Edx).
 
 ### Terminology
 
 freeCodeCamp uses the following terms:
 
-Certification = 'superblock'
+`Certification = 'superblock'`
+
+`Course(s) = 'blocks'`
 
-Course(s) = 'blocks'
+Everything under a course/block is a `'challenge'`
 
-Everything under a course/block is a 'challenge'
+### API
 
----
+This project uses multiple APIs from freeCodeCamp. Check out the following and familiarize yourself.
 
-### Tech stack learning resources
+[Superblock API](https://www.freecodecamp.org/curriculum-data/v1/available-superblocks.json)
 
-Testing with jest
-https://jestjs.io/docs/snapshot-testing
+[Blocks API](https://www.freecodecamp.org/curriculum-data/v1/2022/responsive-web-design.json)
 
-Next.js fullstack framework
-https://nextjs.org/docs/pages/building-your-application/optimizing/testing#jest-and-react-testing-library
-https://dillionmegida.com/p/nextjs-main-concepts/
-https://blog.devgenius.io/advanced-next-js-concepts-8439a8752597
+Mock API: Student information - this will be replaced will real student information in the future. It is **not available publicly** yet, you will need to start the application and go to http://localhost:3001/data.
 
-Next.js terminology:
+### Tech Stack Learning Resources
 
-SSR - Server Side Rendering
+#### Jest Snapshot Testing
 
-SSG - Static Site Generation
+> We are always in need of more tests! Checkout Jest's [snapshot testing docs](https://jestjs.io/docs/snapshot-testing). We recommend learning snapshot testing as we require tests for each component inside our project.
 
-ISR - Incremental Static Regeneration
+#### Next.js Fullstack Framework
 
-CSR - Client Side Rendering
+> This project utilizes Next.js to optimize performance. We recommend learning [main concepts of Next.js](https://dillionmegida.com/p/nextjs-main-concepts/) and familiarizing yourself on [testing with Next.js](https://nextjs.org/docs/pages/building-your-application/optimizing/testing#jest-and-react-testing-library).
 
-SSR is probably the focus point.
+#### Next.js Terminology:
 
-https://dev.to/mbaljeetsingh/what-is-csr-ssr-ssg-isr-different-rendering-strategies-and-which-framework-does-it-better-angular-react-vue-4lkp
+| Term | Value                           |
+| ---- | ------------------------------- |
+| SSR  | Server Side Rendering           |
+| SSG  | Static Site Generation          |
+| ISR  | Incremental Static Regeneration |
+| CSR  | Client Side Rendering           |
 
----
+> SSR is probably the focus point. Checkout this [article](https://dev.to/mbaljeetsingh/what-is-csr-ssr-ssg-isr-different-rendering-strategies-and-which-framework-does-it-better-angular-react-vue-4lkp) to understand these concepts and how they are helpful.
```

Comment:
Maybe remove 'SSR is probably the focus point.' After further working in this repository - it seems that SSR was the focus point, however, the other types of rendering can be used in future changes, and maybe some attention is needed towards the use of ISR.